app.js

```diff
@@ -38,7 +38,7 @@ hamburger.addEventListener('click', function () {
         each.style.textDecoration = 'none';
         each.style.color = 'white';
         each.style.fontWeight = '700';
-        each.style.fontSize = '2.5rem';
+        each.style.fontSize = '2.0rem';
        })
 
 
```

Comment:
- [ ] Your Readme file is not descriptive and professional enough :fearful: Kindly check all these sections(**Table of contents, Title, Descriptions, Tech Stack, Key Features, Future Features, Getting Started, Authors, Contributing, Show your support, Acknowledgment, MIT Link**) in your readme :+1:

**Note: I am giving you a sample readme file to verify your [readme](https://github.com/yash244466666/math-magicians/blob/dev/README.md) file :blush:**